tests/integration/targets/mount/tasks/main.yml

```diff
@@ -294,3 +294,37 @@
     - /tmp/myfs.img
     - /tmp/myfs
     when: ansible_system in ('Linux')
+
+- name: Block to test boot option for Linux
+  block:
+  - name: Create empty file
+    command: dd if=/dev/zero of=/tmp/myfs.img bs=1048576 count=20
+  - name: Format FS
+    community.general.system.filesystem:
```

Comment:
Is there something special with this module, that is the only one to be called by its FQCN ? BTW, `community.general.filesystem` is enough.

Reply:
This task uses the following task from the previous section which was written for the same purpose:

https://github.com/ansible-collections/ansible.posix/blob/main/tests/integration/targets/mount/tasks/main.yml#L232-L236

I'll modify both tasks from `community.general.system.filesystem` to `community.general.filesystem`. Thanks!